packages/clerk-js/src/core/fapiClient.ts

```diff
@@ -136,6 +149,7 @@ export default function createFapiClient(clerkInstance: Clerk): FapiClient {
   function buildEmailAddress(localPart: string): string {
     return buildEmailAddressUtil({
       localPart,
+      // ❓: Is proxyUrl needed here, probably not, just to confirm ?
```

Comment:
❓: Is proxyUrl needed here, probably not, just to confirm ?

Reply:
No. This affects the default support email address that is also configuration from Dashboard > Settings page.